tests/ip/test_mgmt_ipv6_only.py

```diff
@@ -47,3 +64,29 @@ def test_image_download_ipv6_only(creds, duthosts, enum_dut_hostname,
 def test_syslog_ipv6_only(rand_selected_dut, dummy_syslog_server_ip_a, dummy_syslog_server_ip_b,
                           check_default_route, convert_and_restore_config_db_to_ipv6_only): # noqa F811
     run_syslog(rand_selected_dut, dummy_syslog_server_ip_a, dummy_syslog_server_ip_b, check_default_route)
+
+
+def test_ro_user_ipv6_only(localhost, duthosts, enum_rand_one_per_hwsku_hostname, tacacs_creds, check_tacacs_v6,
+                           convert_and_restore_config_db_to_ipv6_only): # noqa F811
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    dutipv6 = get_mgmt_ipv6(duthost)
+
+    if not dutipv6:
+        pytest.skip("Skip IPv6 only test, DUT has no mgmt IPv6.")
```

Comment:
>skip [](http://example.com/codeflow?start=15&length=4)

`convert_and_restore_config_db_to_ipv6_only` will guarantee that there is ipv6. Do now conditional skip. @yejianquan Could you confirm?

Reply:
removed mgmt ipv6 skip